src/js/node/Process.hx

```diff
@@ -349,3 +364,11 @@ typedef MemoryUsage = {
 	heapTotal:Float,
 	heapUsed:Float
 }
+
+typedef Release = {
```

Comment:
Maybe fields other than `name` should be made optional?
> In custom builds from non-release versions of the source tree, only the name property may be present. The additional properties should not be relied upon to exist.

_also, whitespace issue_

Reply:
At the very least `lts` should be made optional or nullable.

Reply:
done